src/Cronofy/CronofyOAuthClient.cs

```diff
@@ -179,12 +179,7 @@ public void RevokeAuthorization(RevokeAuthorizationOptions options)
 
             request.SetJsonBody(requestBody);
 
-            var response = this.HttpClient.GetResponse(request);
```

Comment:
According to [the docs](https://docs.cronofy.com/developers/api/authorization/revoke/#example-response), this request returns a `200`, which is incompatible with [GetValidResponse](https://github.com/cronofy/cronofy-csharp/blob/6c4756e19708a8af080a83f1edc32567cd0aaa9b/src/Cronofy/IHttpClientExtensions.cs#L75) because it will fall into the `case` statements default of throwing a generic exception. GetValidResponse checks `if (response.Code / 100 == 2)` else we fall into throwing exceptions, so a `200` would fall into the exceptions path.

We need to leave this example unchanged.